spec/fabricators/player_stat_fabricator.rb

```diff
@@ -0,0 +1,5 @@
+Fabricator(:player_stat) do
+  player_id { |attrs| attrs[:player].id }
```

Comment:
don't understand these three lines, some foreign methods to me

Reply:
doing it this way will require you pass the `player`, `stat`, and `match`. You can also do it like this: 

```
Fabricator(:player_stat) do
  player
  match
  stat
end
```

and that will allow to pass the attributes and if you don't it will fabricate them for you.